src/infra/src/table/migration/mod.rs

```diff
@@ -28,6 +31,9 @@ impl MigratorTrait for Migrator {
             Box::new(m20241114_000001_create_folders_table::Migration),
             Box::new(m20241115_150000_populate_folders_table::Migration),
             Box::new(m20241116_000001_delete_metas::Migration),
+            Box::new(m20241119_000001_create_dashboards_table::Migration),
+            Box::new(m20241119_000002_populate_dashboards_table::Migration),
+            // Box::new(m20241119_000003_delete_metas::Migration),
```

Comment:
TODO: uncomment

Reply:
please don't forget after testing and before merging :p